src/main.ts

```diff
@@ -247,7 +247,27 @@ export default class CodeEditorShortcuts extends Plugin {
       editorCallback: (editor) =>
         withMultipleSelections(editor, navigateLine, {
           ...defaultMultipleSelectionOptions,
-          args: 'up',
+          args: 'prev',
+        }),
+    });
+
+    this.addCommand({
+      id: 'goToFirstLine',
+      name: 'Go to first line',
+      editorCallback: (editor) =>
+        withMultipleSelections(editor, navigateLine, {
+          ...defaultMultipleSelectionOptions,
+          args: 'top',
+        }),
+    });
+
+    this.addCommand({
+      id: 'goToLastLine',
+      name: 'Go to last line',
```

Comment:
Could you also add these to the table in the README please?

Reply:
I'd forgotten about it! I'll add it.

Reply:
Added